pypeit/reduce.py

```diff
@@ -733,11 +859,74 @@ def __init__(self, sciImg, spectrograph, tslits_dict, par, tilts, **kwargs):
             slit_spat_pos = edgetrace.slit_spat_pos(self.tslits_dict)
             self.order_vec = self.spectrograph.order_vec(slit_spat_pos)
 
+    def get_platescale(self, iord):
```

Comment:
I think the iord should be optional here, so that you can call it without the argument for multislit.

Reply:
Well if we make it require in one and optional in the other than
the Parent call is different for at least one.  A general no-no.
Thoughts?

Reply:
now passing `specobj`